src/amo/components/ShowMoreCard/index.js

```diff
@@ -23,20 +23,46 @@ type UIStateType = {|
   readMoreExpanded: boolean,
 |};
 
+export const truncateToMaxHeight = ({
```

Comment:
This function was moved outside of the component's definition so that it can be the default value for `_truncateToMaxHeight` inside the component.

Reply:
mmm, but just moving it outside the component changes:
```
truncateToMaxHeight: (contents: HTMLElement | null) => void = (
    contents: HTMLElement | null,
  ) =>
```
to
```
export const truncateToMaxHeight = ({
  contents,
  maxHeight,
  setUIState,
  uiState,
}: {|
  contents: HTMLElement | null,
  maxHeight: number,
  setUIState: ($Shape<UIStateType>) => void,
  uiState: UIStateType,
|}) =>
```
but he functionality hasn't changed?

Reply:
Correct, the functionality has not changed. We now need to accept arguments for `maxHeight`, `setUIState` and `uiState` because previously those were being read from `this.props`, but `this.props` doesn't exist when we move the function outside of the component.